src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -906,6 +902,13 @@ fun transformToApprovalLicenceSummary(
   isDueToBeReleasedInTheNextTwoWorkingDays = isDueToBeReleasedInTheNextTwoWorkingDays,
 )
 
+fun getVersionOf(licence: Licence): Long? = when (licence) {
+  is CrdLicence -> licence.versionOfId
+  is HdcLicence -> licence.versionOfId
+  is PrrdLicence -> licence.versionOfId
+  else -> null
+}
+
```

Comment:
There seems to be issing logic for HDC
Could put this in licence,.. hmm but maybe not
not sure this should be in the ToModelTransformers